PactConsumerSwiftTests/MatcherSpec.swift

```diff
@@ -66,7 +66,7 @@ class MatcherSpec: QuickSpec {
       }
 
       it("allows min to be specified") {
-        subject = Matcher.eachLike(arrayItem, min: 4)
+        subject = Matcher.eachLike(arrayItem, minValue: 4)
 
```

Comment:
Sure. Min 4 character var names rule will exclude the `min` and `max` var names.